data_engineering_weekly_data_founders.json

```diff
@@ -0,0 +1,21 @@
to be able to compare companies against each other. I also have personal domain experience. A couple years ago I took a job at a company where I experienced racial discrimination. I realized then that HR works for the company, not the employees. I know people need what I am making because I am one of those people. A study done by a UK company called the vault highlighted the problem as Trust gap. The study found 37 percent of US office workers believed their organization would brush aside at least one form of workplace misconduct if it was likely to impact profit or reputation. What was more troubling was 30 percent of US HR and compliance DMs surveyed believed their organization would look to ignore at least one form of workplace misconduct if it was likely to impact profits or reputation.One of the reasons I know people need what I am making is because of the moment we are living in. With covid and the Great Resignation people are finally sick of putting up with being treated unfairly. They are tired of being discriminated against at work and have decided there are better opportunities out there then the ones they are living with now. An MIT study found that toxic culture in the workplace was driving the great resignation and that it was not just about not pay increase.Another reason why the moment is now is because before the murder of George Floyd it was a taboo topic to discuss Race and discrimination. The frustrating part is it's still not easy navigate on the current solutions like Glassdoor, Teamblind, and Fishbowl. These websites cover racial discrimination but more so as an after thought.Doing research on corporations and the company culture is difficult and time consuming.I know before I take a job I do my research on the company. I usually start on the companies website and look for their diversity and inclusion page to see if they care about diversity. Then I check the reviews to see what people are saying about the company but usually there is not too much race related reviews.I know Glassdoor failed me when I did my research on the company I mentioned above.I recently read a story from the Washington Post that helped me understand I was not alone and this research was common place for people of color. It described the lengthy process people of color started to take to vet a company before accepting a position.With Subtle we are  creating a one stop shop that captures each part of the research phase a person with a diverse background might take. First the corporate news they would find when googling a company. Second, the diversity and workforce metrics a employment seeker would find when going to a company's website.This part of the process can be tricky because without a baseline it's hard to really tell how the company being researched is doing against others in the space. Finally the employment seeker would check the reviews on Glassdoor but if the reviews don’t explicitly say racism or discrimination they can be hard to find.",
```

Comment:
I've updated the guidelines for the contribution here.

```
The founder's story will be published every Wednesday as a separate edition.
Open a pull request with a simple .md file with a title and a one-pager about what you're working on.
You could also point to a blog post you already published, given that the data engineering weekly can republish the content.
```

What do you think of converting the summary content into a markdown file so that I can carry the same format you intend to share with the audience?